### DISCUSSION TITLE
Does PyG have GNN Quantization Support?

### DISCUSSION THREAD

**Discussion by Salmaafifi98** [pyg-team/pytorch_geometric#6133] [Q&A] [answered]
Does PyG support GNN quantization or is there a way to use pytorch's dynamic quantization API for example, with the PyG layers to train/run an 8-bit or 16-bit quantized GNN model?

Thank you!

**Comment by rusty1s** [comment, marked as answer]
I haven't tested this by myself, but I don't see any reason why this shouldn't be supported. Let me know if you have interest in confirming this by adding a test to PyG :)

**Reply by Salmaafifi98** [reply]
It worked and I used it in our recently published work in IEEE/ACM CASES (ESWEEK): https://arxiv.org/abs/2307.01782. Thank you so much, this library is great!
